community/graph-matching/src/test/java/matching/TestMatchingOfCircularPattern.java

```diff
@@ -98,9 +100,12 @@ public static void setupGraph()
     {
         user = graphdb.createNode();
         Node user1 = graphdb.createNode(), user2 = graphdb.createNode(), user3 = graphdb.createNode();
-        user.createRelationshipTo( user1, withName( "FOLLOWS" ) );
-        user1.createRelationshipTo( user3, withName( "FOLLOWS" ) );
-        user.createRelationshipTo( user2, withName( "FOLLOWS" ) );
+        //        return new DynamicRelationshipType( name );
+        user.createRelationshipTo( user1, RelationshipType.withName( "FOLLOWS" ) );
+        //        return new DynamicRelationshipType( name );
+        user1.createRelationshipTo( user3, RelationshipType.withName( "FOLLOWS" ) );
+        //        return new DynamicRelationshipType( name );
+        user.createRelationshipTo( user2, RelationshipType.withName( "FOLLOWS" ) );
```

Comment:
I suspect they are coming from a method you inlined to fix all calls to `DynamicRelationshipType.withName()`.